lib/ansible/modules/windows/win_package.ps1

```diff
@@ -1259,23 +1259,58 @@ $result = @{
     changed = $false
 }
 
-$path = Get-Attr -obj $params -name path -failifempty $true -resultobj $result
-$name = Get-Attr -obj $params -name name -default $path
-$productid = Get-Attr -obj $params -name productid
-if ($productid -eq $null)
+$path = Get-AnsibleParam -obj $params -name "path" -failifempty $true -resultobj $result
+$name = Get-AnsibleParam -obj $params -name "name" -default $path
+$productid = Get-AnsibleParam -obj $params -name "productid" -aliases "product_id"
+$arguments = Get-AnsibleParam -obj $params -name "arguments"
+$options = Get-AnsibleParam -obj $params -name "options"
+#$normalizeargs = Get-AnsibleParam -obj $params -name "normalize_arguments" -type "bool" -aliases "parse_arguments"
+$ensure = Get-AnsibleParam -obj $params -name "state" -default "present" -aliases "ensure"
+$username = Get-AnsibleParam -obj $params -name "user_name"
+$password = Get-AnsibleParam -obj $params -name "user_password"
+$return_code = Get-AnsibleParam -obj $params -name "expected_return_code" -type "int" -default 0
+
+
+if (($arguments -ne $null) -and ($options -ne $null))
 {
-    #Alias added for backwards compat.
-    $productid = Get-Attr -obj $params -name product_id -failifempty $true -resultobj $result
+    fail-json "specify either arguments or options"
```

Comment:
Should be `Fail-Json $result "specify either arguments or options"` / did we just want to overwrite `argument` with `options` and just display a deprecation warning + add to the docs?